src/DonationForms/Routes/ValidationRoute.php

```diff
@@ -34,12 +36,18 @@ public function __invoke(array $request): bool
 
         try {
             $response = $formData->validate();
-            
+
             $this->handleResponse($response);
         } catch (DonationFormFieldErrorsException $exception) {
             $type = DonationFormErrorTypes::VALIDATION;
             $this->logError($type, $exception->getMessage(), $formData);
             $this->sendJsonError($type, $exception->getError());
+        } catch (DonationFormForbidden $exception) {
+            wp_die($exception->getMessage(), 403);
+        } catch (Exception $exception) {
```

Comment:
This resolves an issue with the front-end receiving false positives when hitting the validation endpoint.  Some fields/hooks were throwing an exception that we were not catching here and in some cases was continuing the logic when it shouldn't have been.